src/components/Home.tsx

```diff
@@ -0,0 +1,19 @@
+import { useSelector } from 'react-redux';
+
+interface User {
+    username: string;
+}
+
+interface UserState {
```

Comment:
This is a duplication of the interface from the slice file, any reason not to import it from the slice file instead (or maybe creating another file that only has the type and importing that one in the slice file and in this file)?